src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -583,6 +583,18 @@ internal static byte[] ConvertIntToByteArray(int dwInput)
             return rgbOutput;
         }
 
+        internal static int ConvertByteArrayToInt(byte[] input)
```

Comment:
I think the spec is saying they're arbitrarily-large integers (like `BigInteger`) but that's just my initial reading of it:

https://www.w3.org/TR/xmldsig-core/#sec-CryptoBinary